frontend/styles/var/mixins.scss

```diff
@@ -36,3 +36,17 @@ $max-width: 2560px;
   margin: 0;
   padding: $pad-xxlarge 0 54px 0;
 }
+
+@mixin grey-badge {
+  background-color: $ui-fleet-black-25;
+  color: $core-white;
+  font-size: $xx-small;
+  font-weight: 700;
+  padding: 0 4px;
+  border-radius: 4px;
+  position: relative;
+  top: -2px;
```

Comment:
Can we omit this? It seems to misalign the icon:

with `top: -2px`:
<img width="193" alt="Screenshot 2023-03-30 at 12 08 06 PM" src="https://user-images.githubusercontent.com/61553566/228939283-6db63ab9-c8b7-41a1-84c1-bf1b599c39df.png">

without it:
<img width="193" alt="Screenshot 2023-03-30 at 12 08 25 PM" src="https://user-images.githubusercontent.com/61553566/228939374-2e2e8714-3745-41ed-835f-4e2bfa21aa18.png">